xdp-dump/test-xdpdump.sh

```diff
@@ -0,0 +1,252 @@
+#
+# Test scrip to do basic xdpdump checks
+#
+# shellcheck disable=2039
+#
+XDPDUMP=./xdpdump
+XDP_LOADER=../xdp-loader/xdp-loader
+ALL_TESTS="test_help test_interfaces test_capt_pcap test_capt_term test_exitentry test_snap test_none_xdp"
+
+print_result()
+{
+    echo "$RESULT"
```

Comment:
Please either declare this as a global variable, or just pass it in as a parameter?